### ISSUE TITLE
Garbled Unicode symbols in Windows console

### ISSUE THREAD

**Issue by gumb0** [ethereum/aleth#5236]
Some Unicode symbols look incorrect, for example abridged addresses and Node IDs:
```
INFO  08-31 13:45:41 main net    Id: ##9e75fe97ΓÇª
```
On Linux it looks like:
```
INFO  2018-08-30 09:25:11 aleth net Id: ##581ea427…
```
(This string is constructed here: https://github.com/ethereum/aleth/blob/fdbb63a668148b7a322bdb4a2808872ca43bf357/libdevcore/FixedHash.h#L123 )

**Comment by gumb0** [comment]
Try solution suggested in https://github.com/ethereum-mining/ethminer/issues/197

**Comment by koderproxy** [comment]
i think i am comfortable with  this issue .can i take this issue and try to overcome this problem

**Comment by gumb0** [comment]
@koderproxy Sure, go ahead.

**Comment by halfalicious** [comment]
Fixed by #5367
